TriblerGUI/widgets/manageplaylistpage.py

```diff
@@ -1,3 +1,6 @@
+from __future__ import absolute_import
+
+from six.moves import xrange
```

Comment:
Please put this after the `PyQt5` import and add a newline between the `PyQt5` imports and the `TriblerGUI` imports.